### PR TITLE
LCI 15 cards

### PR DESCRIPTION
**Pull request by Simisays**
Fabrication Foundry need some more work

### CODE REVIEW

File: forge-gui/res/cardsfolder/upcoming/abyssal_gorestalker.txt [hunk 1]
```diff
@@ -0,0 +1,9 @@
+Name:Abyssal Gorestalker
+ManaCost:4 B B
+Types:Creature Horror
+PT:6/6
+DeckHas:Ability$Sacrifice
```
Comment (tool4ever) [inline forge-gui/res/cardsfolder/upcoming/abyssal_gorestalker.txt:5]: doubled

File: forge-gui/res/cardsfolder/upcoming/fabrication_foundry.txt [hunk 2]
```diff
@@ -0,0 +1,9 @@
+Name:Fabrication Foundry
+ManaCost:1 W
+Types:Artifact
+A:AB$ Mana | Cost$ T | Produced$ W | Amount$ 1 | RestrictValid$ Spell.Artifact,Activated.Artifact+inZoneBattlefield | SpellDescription$ Add {W}. Spend this mana only to cast an artifact spell or activate an ability of an artifact source.
```
Comment (tool4ever) [inline forge-gui/res/cardsfolder/upcoming/fabrication_foundry.txt:4]: "source"

File: forge-gui/res/cardsfolder/upcoming/fabrication_foundry.txt [hunk 3]
```diff
@@ -0,0 +1,9 @@
...
+A:AB$ ChangeZone | Cost$ 2 W T Exile<X/Artifact.Other/other artifacts> | Origin$ Graveyard | Destination$ Battlefield |ValidTgts$ Artifact.cmcLEX+YouOwn | TgtPrompt$ Select target artifact card with total mana value X | SorcerySpeed$ True | SpellDescription$ Return target artifact card with mana value X or less from your graveyard to the battlefield. Activate only as a sorcery.
```
Comment (tool4ever) [inline forge-gui/res/cardsfolder/upcoming/fabrication_foundry.txt:5]: what about X=0?

Comment (tool4ever) [reply forge-gui/res/cardsfolder/upcoming/fabrication_foundry.txt:5]: the way X gets defined here as an open ended sum seems very tricky
@Northmoc 
any idea?

File: forge-gui/res/cardsfolder/upcoming/ojer_kaslem_deepest_growth_temple_of_cultivation.txt [hunk 4]
```diff
@@ -0,0 +1,22 @@
...
+AlternateMode:DoubleFaced
+Oracle:Trample\nWhenever Ojer Kaslem deals combat damage to a player, reveal that many cards from the top of your library. You may put a creature card and/or a land card from among them onto the battlefield. Put the rest on the bottom in a random order.\nWhen Ojer Kaslem dies, return it to the battlefield tapped and transformed under its owner's control.
+
+ALTERNATE
+
+Name:Temple of Cultivation
+ManaCost:no cost
+Types:Land
+A:AB$ Mana | Cost$ T | Produced$ G | SpellDescription$ Add {G}.
+A:AB$ SetState | Cost$ 2 G T | CheckSVar$ X | SVarCompare$ GE10 | Defined$ Self | Mode$ Transform | SorcerySpeed$ True | SpellDescription$ Transform CARDNAME. Activate only if you control ten or more permanents and only as a sorcery.
```
Comment (tool4ever) [inline forge-gui/res/cardsfolder/upcoming/ojer_kaslem_deepest_growth_temple_of_cultivation.txt:20]: use IsPresent

File: forge-gui/res/cardsfolder/upcoming/spyglass_siren.txt [hunk 5]
```diff
@@ -0,0 +1,8 @@
+Name:Spyglass Siren
+ManaCost:U
+Types:Creature Siren Pirate
+PT:1/1
+T:Mode$ ChangesZone | Origin$ Any | Destination$ Battlefield | ValidCard$ Card.Self | Execute$ TrigToken | TriggerDescription$ Whenever CARDNAME enters the battlefield or attacks, create a Map token. (It's an artifact with "{1}, {T}, Sacrifice this artifact: Target creature you control explores. Activate only as a sorcery.")
```
Comment (tool4ever) [inline forge-gui/res/cardsfolder/upcoming/spyglass_siren.txt:5]: wrong description

File: forge-gui/res/cardsfolder/upcoming/subterranean_schooner.txt [hunk 6]
```diff
@@ -0,0 +1,16 @@
...
+Types:Artifact Vehicle
+PT:3/4
+K:Crew:1
+T:Mode$ BecomesCrewed | ValidVehicle$ Card.Self | Execute$ RememberCrew | Static$ True
+SVar:RememberCrew:DB$ PumpAll | ValidCards$ Creature.TriggeredCrew | RememberAllPumped$ True
+T:Mode$ ChangesZone | Origin$ Battlefield | Destination$ Any | ValidCard$ Card.Self | Execute$ DBCleanup | Static$ True
+T:Mode$ Phase | Phase$ End of Turn | ValidPlayer$ Player | TriggerZones$ Battlefield | Execute$ DBCleanup | Static$ True
+SVar:DBCleanup:DB$ Cleanup | ClearRemembered$ True
+T:Mode$ Attacks | ValidCard$ Card.Self | TriggerZones$ Battlefield | Execute$ TrigExplore | TriggerDescription$ Whenever CARDNAME attacks, target creature that crewed it this turn explores. (Reveal the top card of your library. Put that card into your hand if it's a land. Otherwise, put a +1/+1 counter on that creature, then put the card back or put it into your graveyard.)
+SVar:TrigReturn:DB$ ChangeZone | ValidTgts$ Creature.IsRemembered | TargetMin$ 0 | TargetMax$ 1 | Origin$ Battlefield | Destination$ Hand | TgtPrompt$ Select up to one target creature to return to its owner's hand
```
Comment (tool4ever) [inline forge-gui/res/cardsfolder/upcoming/subterranean_schooner.txt:12]: unused

File: forge-gui/res/cardsfolder/upcoming/vanguard_of_the_rose.txt [hunk 7]
```diff
@@ -0,0 +1,8 @@
+Name:Vanguard of the Rose
+ManaCost:1 W
+Types:Creature Vampire Knight
+PT:3/1
+A:AB$ Pump | Cost$ Sac<1/Creature.Other;Artifact/another creature or artifact> | Defined$ Self | KW$ Indestructible | SubAbility$ DBTap | SpellDescription$ Sacrifice another creature or artifact: CARDNAME gains indestructible until end of turn. Tap it.
```
Comment (tool4ever) [inline forge-gui/res/cardsfolder/upcoming/vanguard_of_the_rose.txt:5]: tidy description

Comment (tool4ever) [inline forge-gui/res/cardsfolder/upcoming/vanguard_of_the_rose.txt:5]: both are Other

File: forge-gui/res/cardsfolder/upcoming/waterwind_scout.txt [hunk 8]
```diff
@@ -0,0 +1,8 @@
+Name:Waterwind Scout
+ManaCost:2 U
+Types:Creature Merfolk Scout
+PT:2/2
+T:Mode$ ChangesZone | Origin$ Any | Destination$ Battlefield | ValidCard$ Card.Self | Execute$ TrigToken | TriggerDescription$ Whenever CARDNAME enters the battlefield or attacks, create a Map token. (It's an artifact with "{1}, {T}, Sacrifice this artifact: Target creature you control explores. Activate only as a sorcery.")
```
Comment (tool4ever) [inline forge-gui/res/cardsfolder/upcoming/waterwind_scout.txt:5]: wrong description

File: forge-gui/res/cardsfolder/upcoming/fabrication_foundry.txt (hunk 3 above)
Comment (Simisays) [reply forge-gui/res/cardsfolder/upcoming/fabrication_foundry.txt:5]: I'll remove this one for now I think

File: forge-gui/res/cardsfolder/upcoming/ojer_kaslem_deepest_growth_temple_of_cultivation.txt [hunk 9]
```diff
@@ -0,0 +1,21 @@
+Name:Ojer Kaslem, Deepest Growth
+ManaCost:3 G G
+Types:Legendary Creature God
+PT:6/5
+K:Trample
+T:Mode$ DamageDone | ValidSource$ Card.Self | ValidTarget$ Player | Execute$ TrigDig | CombatDamage$ True | TriggerDescription$ Whenever NICKNAME deals combat damage to a player, reveal that many cards from the top of your library. You may put a creature card and/or a land card from among them onto the battlefield. Put the rest on the bottom in a random order.
+SVar:TrigDig:DB$ DigMultiple | SourceZone$ Library | DigNum$ X | ChangeValid$ Creature,Land | Optional$ True | DestinationZone$ Battlefield | RestRandomOrder$ True
```
Comment (tool4ever) [inline forge-gui/res/cardsfolder/upcoming/ojer_kaslem_deepest_growth_temple_of_cultivation.txt:7]: reveal missing

File: forge-gui/res/cardsfolder/upcoming/ojer_kaslem_deepest_growth_temple_of_cultivation.txt [hunk 10]
```diff
@@ -0,0 +1,21 @@
...
+AlternateMode:DoubleFaced
+Oracle:Trample\nWhenever Ojer Kaslem deals combat damage to a player, reveal that many cards from the top of your library. You may put a creature card and/or a land card from among them onto the battlefield. Put the rest on the bottom in a random order.\nWhen Ojer Kaslem dies, return it to the battlefield tapped and transformed under its owner's control.
+
+ALTERNATE
+
+Name:Temple of Cultivation
+ManaCost:no cost
+Types:Land
+A:AB$ Mana | Cost$ T | Produced$ G | SpellDescription$ Add {G}.
+A:AB$ SetState | Cost$ 2 G T | IsPresent$ Permanent.YouCtrl | SVarCompare$ GE10 | Defined$ Self | Mode$ Transform | SorcerySpeed$ True | SpellDescription$ Transform CARDNAME. Activate only if you control ten or more permanents and only as a sorcery.
```
Comment (tool4ever) [inline forge-gui/res/cardsfolder/upcoming/ojer_kaslem_deepest_growth_temple_of_cultivation.txt:20]: wrong param

**Review by tool4ever — APPROVED** [review]

File: forge-gui/res/cardsfolder/upcoming/the_ancient_one.txt [hunk 11]
```diff
@@ -0,0 +1,14 @@
+Name:The Ancient One
+ManaCost:U B
+Types:Legendary Creature Spirit God
+PT:8/8
+S:Mode$ Continuous | Affected$ Card.Self | AddHiddenKeyword$ CARDNAME can't attack or block. | IsPresent$ Permanent.YouOwn | PresentZone$ Graveyard | PresentCompare$ LE7 | Description$ Descend 8 — CARDNAME can't attack or block unless there are eight or more permanent cards in your graveyard.
+A:AB$ Draw | Cost$ 2 U B | SubAbility$ DBDiscard | SpellDescription$ Draw a card, then discard a card. When you discard a card this way, target player mills cards equal to its mana value.
+SVar:DBDiscard:DB$ Discard | Defined$ You | Mode$ TgtChoose |  NumCards$ 1 | RememberDiscarded$ True | SubAbility$ DBImmediateTrig | StackDescription$ None
```
Comment (Northmoc) [inline forge-gui/res/cardsfolder/upcoming/the_ancient_one.txt:7]: spacing

Comment (Northmoc) [inline forge-gui/res/cardsfolder/upcoming/the_ancient_one.txt:7]: I bet this StackDescription/SpellDescription needs tweaking for the stack

File: forge-gui/res/cardsfolder/upcoming/vanguard_of_the_rose.txt [hunk 12]
```diff
@@ -0,0 +1,8 @@
+Name:Vanguard of the Rose
+ManaCost:1 W
+Types:Creature Vampire Knight
+PT:3/1
+A:AB$ Pump | Cost$ Sac<1/Creature.Other;Artifact.Other/another creature or artifact> | Defined$ Self | KW$ Indestructible | SubAbility$ DBTap | SpellDescription$ CARDNAME gains indestructible until end of turn.
+SVar:DBTap:DB$ Tap | Defined$ Self | SpellDescription$ Tap it.
+DeckHas:Ability$Sacrifice
```
Comment (Northmoc) [inline forge-gui/res/cardsfolder/upcoming/vanguard_of_the_rose.txt:7]: DeckHints:Ability$Token & Type$Artifact|Treasure|Map etc?

File: forge-gui/res/cardsfolder/upcoming/vanguard_of_the_rose.txt [hunk 13]
```diff
@@ -0,0 +1,8 @@
+Name:Vanguard of the Rose
+ManaCost:1 W
+Types:Creature Vampire Knight
+PT:3/1
+A:AB$ Pump | Cost$ Sac<1/Creature.Other;Artifact.Other/another creature or artifact> | Defined$ Self | KW$ Indestructible | SubAbility$ DBTap | SpellDescription$ CARDNAME gains indestructible until end of turn.
+SVar:DBTap:DB$ Tap | Defined$ Self | SpellDescription$ Tap it.
```
Comment (Northmoc) [inline forge-gui/res/cardsfolder/upcoming/vanguard_of_the_rose.txt:6]: Probably needs StackDescription$ SpellDescription
Line 5 might also need that... sometimes Pump is fine though

**Review by tool4ever — APPROVED** [review]

**Pull request merged**